src/rules/property-no-unknown/README.md

```diff
@@ -85,3 +87,20 @@ The following patterns are *not* considered warnings:
   custom: 10px;
 }
 ```
+### `checkPrefixed: true | false` (default: `false`)
+
+This option forces to check vendor prefixed properties.
```

Comment:
How about: If `true`, this rule will check vendor-prefixed properties.

Please explain why this is not enabled by default, and why a user would want to enable it.